be/src/storage/tablet_reader.cpp

```diff
@@ -89,6 +89,7 @@ Status TabletReader::prepare() {
             LOG(WARNING) << ss.str();
             return Status::InternalError(ss.str().c_str());
         }
+        RETURN_IF_ERROR(_init_delete_predicates(&_delete_predicates));
```

Comment:
It will be better to add some comments here to tell others why the _delete_predicates should be initiated here.